integration_tests/integration/prisonerProfile.cy.ts

```diff
@@ -106,5 +106,12 @@ context('Prisoner profile page', () => {
       })
     prisonerProfilePage.visitTabVisitStatus().eq(0).contains('Booked')
     prisonerProfilePage.visitTabViewFullHistory().contains('View full visits history')
+    prisonerProfilePage
```

Comment:
Testing here is OK, but probably worth making sure it's in the unit tests too. E.g. `./server/routes/prisoner.test.ts` has a couple of tests that reference `data-test=view-dps-profile` so could check the URL there.